### ISSUE TITLE
Why my tf-lite model's fullyconnect and softmax options not be delegated with CoreML?

### ISSUE THREAD

**Issue by xunbing** [tensorflow/tensorflow#51626]
*enviorment: iPhone 12*

I have a model converted by TFLiteConverter. It has some opttions like lstm、add、mul、fullyconnect、softmax. When I use the CoreML delegate like the example code in docs. I found that There are just some of the all options delegated by CoreML. I understand some option like lstm would not be delegated. But I have no idea when I found my fullyconnect and softmax are also not be delegated by profiling the app in Xcode Instrument. 
For contrast, I download a DensetNet model. I found CoreML delegate almost all the options include fc and softmax.

Why is that? Any reply will be appreciated.

**Comment by xunbing** [comment]
Now I found the reason why my Dense Not been delegate to CoreML is that its' dims except last dim are not 1.

**Comment by Ram-WD** [comment]
> Now I found the reason why my Dense Not been delegate to CoreML is that its' dims except last dim are not 1.

what do u mean ? i am having the same issue , 

Error compiling model compiler error: Size of bias = 0 is not equal to the output channels = 7 in layer FullyConnectedOpBuilder_17
